Source/CesiumRuntime/Private/CesiumPointAttenuationVertexFactory.cpp

```diff
@@ -13,7 +13,12 @@
 #include "MaterialDomain.h"
 #endif
 
+#if ENGINE_MAJOR_VERSION > 5 || ENGINE_MINOR_VERSION >= 3
```

Comment:
I think this #if works, but my brain had to pause a bit and work out the logic.

What do you think of this?
```
#define ENGINE_VERSION_5_3_OR_HIGHER   (ENGINE_MAJOR_VERSION > 5 || ENGINE_MINOR_VERSION >= 3)
...
#if ENGINE_VERSION_5_3_OR_HIGHER   
  ... some code
```

Reply:
I also was confused by this when I reviewed a PR with this kind of check. But ultimately we have a lot of these checks across our files, and if we did this here, we would want to do it across the plugin for consistency. And you'd need a little manage all the `#defines` so that they don't get redefined between files. It seems a little messy 😅

Reply:
For sure. A centralized place for a #define such as this is would be ideal.

Reply:
I don't think we need to mix this in with this PR. If you want to open a separate one to do this plugin-wide, I'm all for it.

Reply:
That works. New issue is here, #1214